README.md

```diff
@@ -189,13 +175,13 @@ Current is D
 </style>
 <div class="wrapper">
  <div id="A" style="order: 2">A
-   <button id="a" style="order: 3">Button A</button>
+   <button id="a" style="order: 3">a</button>
  </div>
  <div id="B" style="order: 3">B
-   <button id="b">Button B</button>
+   <button id="b">b</button>
  </div>
  <div id="C" style="order: 1">C
-   <button id="c">Button C</button>
+   <button id="c">c</button>
  </div>
 </div>
 ```
```

Comment:
Below this point, there are many instances of `next DOM` - those are confusing since there's no such concept in the algorithms. I'm not sure how to fix those up, really. I believe the "next DOM" and "previous DOM" are handled by the existing algorithms in the focus navigation spec, right? If so, perhaps just refer to those sections?

Reply:
Ack. I will open a new PR to handle this.